carta/session.py

```diff
@@ -417,6 +417,43 @@ def clear_raster_scaling_reference(self):
         """Clear the raster scaling reference."""
         self.call_action("clearRasterScalingReference")
 
+    # VIEWER MODES
+    @validate(Boolean())
+    def set_viewer_mode(self, multiple=True):
+        """
+        Switch between single-panel mode and multiple-panel mode.
+
+        Parameters
+        ----------
+        multiple : {0}
+            Whether to use multiple-panel mode. Default is ``True``.
+        """
+        self.call_action("widgetsStore.setImageMultiPanelEnabled", multiple)
+
+    def previous_page(self):
+        """Go to previous page in viewer."""
+        self.call_action("widgetsStore.onPreviousPageClick")
+
+    def next_page(self):
+        """Go to next page in viewer."""
+        self.call_action("widgetsStore.onNextPageClick")
+
+    @validate(Number(), Number())
```

Comment:
validate columns/rows a a number is not sufficient. Try to input 0 or -1 to see what happens. These two arguments should be positive integer.

Reply:
Done with a tricky method in 14f6ca5. I set the validation check to `OneOf(*range(1, 101))` for `columns` and `rows`. I'm not sure if it's possible for user to set up to 100 for columns and rows, but I think it should work. Please let me know if it's a proper way of solving this issue